wire-ios-sync-engine/Source/Use cases/ImportBackupUseCase/ImportBackupUseCase.swift

```diff
@@ -164,12 +165,16 @@ struct ImportBackupUseCase: ImportBackupUseCaseProtocol {
             let outputStream = OutputStream(url: decryptedURL, append: false)
         else { throw ImportBackupError.failedToCreateStreamForDecryption }
 
-        try streamDecryptor.decrypt(
-            input: inputStream,
-            output: outputStream,
-            accountID: accountID,
-            password: password
-        )
+        do {
+            try streamDecryptor.decrypt(
+                input: inputStream,
+                output: outputStream,
+                accountID: accountID,
+                password: password
+            )
+        } catch WireCrypto.ChaCha20Poly1305.StreamEncryption.EncryptionError.mismatchingUUID {
+            throw ImportBackupError.invalidAccountID
+        }
```

Comment:
Fixes an unrelated issue:
Trying to import a backup from another user was failing silently. This fixes the alert presentation.

Reply:
Moved it to another PR: https://github.com/wireapp/wire-ios/pull/2709